### ISSUE TITLE
"LTI Variable Substitutions" documentation incomplete

### ISSUE THREAD

**Issue by lsloan** [instructure/canvas-lms#1356]
### Summary:

On the Canvas documentation website, the page for "LTI Variable Substitutions" is incomplete.  It is cut off just after the "Supported Substitutions".

### Steps to reproduce:

 1. Go to: https://canvas.instructure.com/doc/api/file.tools_variable_substitutions.html
 2. Scroll down to the "Supported Substitutions" heading.

### Expected behavior:

We should see the documentation for supported substitutions.

### Actual behavior:

An empty space is shown where the supported substitutions should appear.

### Additional notes:

I was able to find the complete documentation in this repo, at: https://github.com/instructure/canvas-lms/blob/master/doc/api/tools_variable_substitutions.md

**Comment by meeple142** [comment]
as per lines 1-8 in that markdown file
https://github.com/instructure/canvas-lms/blame/a664cdb0b26bf9d4473c0204dba38fc73a34ece7/doc/api/tools_variable_substitutions.head.md#L1-L8

quoted here
```
<!--
  DONT EDIT THIS FILE DIRECTLY
  Update:
    - doc/api/tools_variable_substitutions.head.md
    - lib/lti/variable_expander.rb
  then run `script/generate_lti_variable_substitution_markdown`
-->
```
Looks like all someone has to do is run the script `script/generate_lti_variable_substitution_markdown`

you can look at their code comments here in the mean time still inconvenient https://github.com/instructure/canvas-lms/blob/a664cdb0b26bf9d4473c0204dba38fc73a34ece7/lib/lti/variable_expander.rb

**Comment by bermudezjd** [comment]
This ticket is a dupe of #1335

**Comment by jonespm** [comment]
This looks to have been fixed since last the last comment in early 2019.

**Comment by westonkd** [comment]
Fixed by [62dd89c](https://github.com/instructure/canvas-lms/commit/62dd89c51789c4541c30b988d2155024b9404a7d)
